src/plugins/GUITestBase/src/GTUtilsMsaEditor.cpp

```diff
@@ -200,7 +203,7 @@ QRect GTUtilsMsaEditor::getColumnHeaderRect(GUITestOpStatus& os, int column) {
     MSAEditor* editor = getEditor(os);
     GT_CHECK_RESULT(nullptr != editor, "MSA Editor is NULL", QRect());
 
-    BaseWidthController* baseWidthController = editor->getUI()->getBaseWidthController();
+    BaseWidthController *baseWidthController = editor->getUI()->getUI()->getBaseWidthController();
```

Comment:
getUI()->getUI() chainer looks ugly. Can we have a better naming here?

Reply:
Have no nice idea.
getTopUI()->getUI()
  or
getUI()->getChildUI()
?